lib/micro/case/result/transitions.rb

```diff
@@ -5,7 +5,7 @@ class Case
     class Result
       class Transitions
         MapEverything = -> (result, use_case_attributes) do
-          result_track = result.success? ? :success : :failure
+          result_track = result.to_sym
```

Comment:
@tomascco what do you think to replace the `result_track` variable by a direct call to `result.to_sym`?

Reply:
Well, I didn't think about it :flushed:. I'll do it.